CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v11.0.1
+* **BUG FIX**
+  * Resolved an EFS mount point issue when using an `access_point_id` in the `authorization_confi` instead of the `root_directory` preventing the
```

Comment:
```suggestion
  * Resolved an EFS mount point issue when using an `access_point_id` in the `authorization_config` instead of the `root_directory` preventing the
```